tests/server/test_resource_list.py

```diff
@@ -162,12 +171,12 @@ async def create_resource(
             )
             await res.insert()
 
-    await create_resource("agent1", "/etc/file1", "std::File", ResourceState.available, [1, 2, 3])
+    await create_resource("agent1", "/etc/file1", "std::File", ResourceState.available, [1, 2, 3, 4])
```

Comment:
This fixture is updated, but the fixture is not used by the test case that was updated?

Reply:
Good catch, this really didn't need to be changed